courses/management/commands/sync_courseruns.py

```diff
@@ -0,0 +1,86 @@
+"""
+Management command to sync grades and certificates for a course run
+"""
+from django.core.management.base import BaseCommand, CommandError
+
+from courses.models import CourseRun
+from courseware.api import get_edx_api_client
+from users.models import User
+from mitxpro.utils import now_in_utc
+
+
+class Command(BaseCommand):
+    """
+    Command to sync grades and certificates for a course run.
+    """
+
+    help = "Sync dates and title for all or a specific course run from edX."
+
+    def add_arguments(self, parser):
+        parser.add_argument(
+            "--run",
+            type=str,
+            help="The 'courseware_id' value for a CourseRun",
+            required=False,
+        )
+        super().add_arguments(parser)
+
+    def handle(self, *args, **options):  # pylint: disable=too-many-locals
+        """Handle command execution"""
+        runs: [CourseRun] = []
+        if options["run"]:
+            try:
+                runs = [CourseRun.objects.get(courseware_id=options["run"])]
+            except CourseRun.DoesNotExist:
+                raise CommandError(
+                    "Could not find run with courseware_id={}".format(options["run"])
+                )
+        else:
+            now = now_in_utc()
+            runs = CourseRun.objects.filter(end_date__gt=now)
+
+        # TODO: This will be updated to use a service worker account
+        user = User.objects.filter(is_superuser=True).first()
```

Comment:
what if superuser does not exist?

Reply:
@umarmughal824 on line `42` I've got the `TODO` comment that this will be replaced with a service worker instead.